v3/catalog-shacl.shce

```diff
@@ -0,0 +1,260 @@
+BASE <#>
```

Comment:
```suggestion
BASE <https://catalog.solidproject.org/shapes>
```

Besides fixing the failing build it will should also minimize chance of tooling determining wrong base or not setting any base at all.

Reply:
Sounds good so long as that is the location this is expected to be published to - @jeff-zucker can you confirm this is the correct base?

Reply:
probably instead of publishing to gh-pages, we can setup context negotiation between Turtle / Compact SHACL / ShEx

Reply:
We probably want to include `v3` in that URL

Reply:
Let's deal with it in a separate Issue/PR